tests/lib.rs

```diff
@@ -199,4 +200,33 @@ mod util {
             }
         }
     }
+
+    /// Without Right::Lookup, looking up paths in a directory is not allowed
+    #[test]
+    fn test_basic_dir_err() {
+        let tdir = tempdir().unwrap();
```

Comment:
nit: might be nice to make this a setup function like `setup_test(Option<Right>)` to avoid duplication of the test setup.